cinema/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```

Comment:
You should register your models here using `admin.site.register(YourModelName)`. This will allow you to manage your models through the Django admin interface.